cli/gumball-machine/templates/addConfigLines.ts

```diff
@@ -0,0 +1,10 @@
+import {
+    AddConfigLinesInstructionArgs
+} from "../../../contracts/sdk/gumball-machine";
+
+export const configLinesToAdd: AddConfigLinesInstructionArgs =  { 
+  newConfigLinesData: Buffer.from(
+    "uluvnpwncgchwnbqfpbtdlcpdthc" + 
+    "aauvnpwncgchwnbqfpbtdlcpdthc"
+  )
+};
```

Comment:
maybe you can allow users to pass in JSONs, and then have `template/addConfigLines.ts` check that the typecast to `AddConfigLinesInstructionArgs` is valid